pytest_remotedata/plugin.py

```diff
@@ -13,9 +12,9 @@ def pytest_addoption(parser):
 
     # The following means that if --remote-data is not specified, the default
     # is 'none', but if it is specified without arguments (--remote-data), it
-    # defaults to '--remote-data=any'.
+    # defaults to '--remote-data=any'. -R is short for --remote-data
     parser.addoption(
-        "--remote-data", nargs="?", const='any', default='none',
+        "--remote-data", "-R", nargs="?", const='any', default='none',
```

Comment:
I didn't check but we had a past problem where we accidentally overrode an existing shortcut from upstream. Are you sure this won't conflict with other stuff?

Reply:
yep, that was one of the other ones. There maybe override with other plugins, but not with core pytest or the ones I have installed:

```
$ ~/munka/devel/astroquery [esasky-doc-cleanup|…1⚑ 19] $ pytest -R
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: -R
  inifile: /Users/bsipocz/munka/devel/astroquery/setup.cfg
  rootdir: /Users/bsipocz/munka/devel/astroquery
  ```
  
  ```
  plugins: anyio-3.6.1, remotedata-0.3.3, requests-mock-1.9.3, rerunfailures-10.2, astropy-header-0.2.2, doctestplus-0.12.1, mock-3.8.2, dependency-0.5.1, astropy-0.10.0, filter-subpackage-0.1.1, openfiles-0.5.0, nbval-0.9.7.dev0, hypothesis-6.54.5, cov-3.0.0, arraydiff-0.5.0
  ```

Reply:
Thanks for checking! We can revisit this in the future if a conflict does arise I hope not. 😸